interface/billing/edi_270.php

```diff
@@ -152,14 +153,17 @@
     $facilities     = getUserFacilities($_SESSION['authUserID']);
 
     // Get the Providers information
-    $providers      = EDI270::getUsernames();
+    $providers      = EDI270EligibilityPartnerService::getUsernames();
 
     //Get the x12 partners information
-    $clearinghouses = EDI270::getX12Partner();
+    $clearinghouses = EDI270EligibilityPartnerService::getX12Partner();
 
     if (isset($_POST['form_xmit']) && !empty($_POST['form_xmit']) && $res) {
         $eFlag = !$GLOBALS['disable_eligibility_log'];
         // make the batch request
+        //This should not be so tightly bound to the EDI270 class
+        //This should be an interface to implement different clearinghouses of 270 requests
+        //EDI270Interface::requestRealTimeEligible($res, $X12info, $segTer, $compEleSep, $eFlag, (new EDI270WayStar()));
```

Comment:
@stephenwaite were you talking about this class?

@juggernautsei if this isn't used should it be removed?

Reply:
@adunsulag I will remove it. I thought I was including an engineering note for the future. I do realize now that an interface can't have a static method. This would have to be changed to a class call. In that class there would have to be an interface injected in the constructor. I will remove my comments.

Reply:
hi @juggernautsei, would create a new issue that states EDI270 restructure that's fixed by this PR and to be consistent with `src/Services` maybe change the directory name to `src/Billing/Services/` here?